test/aws_host_config_test.js

```diff
@@ -34,7 +34,8 @@ suite('configuration/aws', function() {
       workerType: 'ami-333333',
       workerNodeType: 'c3.xlarge',
       workerGroup: 'us-west-2',
-      capacity: 1
+      capacity: 1,
+      publicIp: '22.33.44.252'
```

Comment:
yea, this is hardcoded in the aws_metadata which sort of acts like a mock for the meta-data endpoint.  It could be anything, just as long as it matched what's returned.